tests/testthat/test-bm-write-csv.R

```diff
@@ -0,0 +1,12 @@
+test_that("write_csv benchmark works", {
+  expect_s3_class(
+    run_benchmark(
+      write_csv,
+      source = "nyctaxi_sample",
+      writer = c("arrow", "data.table", "vroom", "readr", "base")
+      ), "arrowbench_results"
+  )
+})
+
+
+
```

Comment:
```suggestion
})

```

Super minor nitpick that lintr would complain about if we had it on here